.github/workflows/run-tests.yml

```diff
@@ -413,7 +413,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: ${{ fromJson(needs.set-up-single-test.outputs.runs_on) }}
```

Comment:
Are we intentionally removing this capability? If so, we should probably remove that whole output and code path.

Reply:
Ah, that's why I didn't put it up for review 2w ago. Okay, I'll need to figure out how to keep the option and provide a default per matrix item then.